be/src/docker-compose.override.yml

```diff
@@ -195,6 +195,9 @@ services:
       - EventBusRetryCount=5
       - SupportedCultures=${SUPPORTED_CULTURES}
       - DefaultCulture=${DEFAULT_CULTURE}
+      - SenderEmail=orders@eltap.com
+      - SenderName=ELTAP
```

Comment:
**suggestion:** let's move all of these to .env and set a variable here. Remember to update README